cdap-ui/app/cdap/services/AlertStatus.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// TO-DO(CDAP-17088): Move this to Alert component once it has been converted to TS
+enum ALERT_STATUS {
+  Success = 'success',
```

Comment:
We can capitalize strings to keep consistency

Reply:
This is coming from the expected props for the Alert component. I agree it looks a little funny, but we use the Alert component in a lot of places that would also need updating so I don't want to make a change to that in this PR. That said, I've noted the suggested improvement in ALERT_STATUS in the JIRA for the future typescript Alert component.